app/views/users/_form.html.erb

```diff
@@ -49,6 +49,13 @@
                     <% end %>
                 </div>
             </div>
+            <h3 class="p3"><%= t('users.daily_summary') %></h3>
```

Comment:
Why `h3`? It is really a header?

Reply:
Yes, it was header, indeed. But as I removed `daily_summary_intention`, it's no longer needed.